torch/_export/converter.py

```diff
@@ -408,7 +412,7 @@ def __init__(
         self.name_to_constant: Dict[str, Any] = name_to_constant
 
         # Mapping from torchscript node output name to attribute fully qualified name
-        self.name_to_attribute_fqn: Dict[str, str] = {}
+        self.name_to_attribute_fqn: Dict[str, str] = name_to_attribute_fqn
```

Comment:
I am starting to loose track of the meaning of

`name_to_param`, `name_to_buffer`, `name_to_node`, `name_to_constant`, `name_to_attribute_fqn`. 

Do you mind adding some comments on so that we don't confuse our future selves?